admin/product.php

```diff
@@ -11,7 +11,7 @@
 $action = (isset($_GET['action']) ? $_GET['action'] : '');
 require(DIR_WS_CLASSES . 'currencies.php');
 $currencies = new currencies();
-$product_type = (isset($_GET['pID']) ? zen_get_products_type($_GET['pID']) : (isset($_POST['product_type']) ? $_POST['product_type'] : 1));
+$product_type = (isset($_POST['product_type']) ? $_POST['product_type'] : (isset($_GET['pID']) ? $_GET['pID'] : 1));

```

Comment:
This should still call `zen_get_products_type()`

Reply:
Correct... Sorry and corrected.